packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -1,33 +1,40 @@
 # Apex Replay Debugger for Visual Studio Code
+
 Apex Replay Debugger simulates a live debugging session using a debug log that is a recording of all interactions in a transaction. You no longer need to parse through thousands of log lines manually. Instead, Apex Replay Debugger presents the logged information similarly to an interactive debugger, so you can debug your Apex code. The debugging process is a repetition of editing your Apex code, pushing or deploying the code to your org, reproducing the buggy scenario, downloading the resulting debug log, and launching Apex Replay Debugger with that debug log.
 
+**For best results, do not install this extension directly. Install the complete [Salesforce Extension Pack] (https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) instead.**
```

Comment:
Why do we really need this ? Are extension dependencies not getting properly downloaded ?

Reply:
Yeah, its actually something we had already on most of the extension, but it was missing on this one. Based on download numbers some extensions seem to be getting installed more than the extension pack which seems weird. Maybe its the way Microsoft counts downloads, but I want to make sure as I have encountered times when customers were confused because they only added one extension instead of the pack.